docs/reference/release-notes/7.0.0.asciidoc

```diff
@@ -1,8 +1,1648 @@
 [[release-notes-7.0.0]]
 == {es} version 7.0.0
 
-//These release notes include all changes made in the alpha, beta, and RC
-//releases of 7.0.0. 
+These release notes include all changes made in the alpha, beta, and RC
+releases of 7.0.0.
 
-//Also see <<breaking-changes-7.0,Breaking changes in 7.0.0>>.
+Also see <<breaking-changes-7.0,Breaking changes in 7.0>>.
 
+[[breaking-7.0.0]]
+[float]
+=== Breaking changes
+
+Aggregations::
+* Remove support for deprecated params._agg/_aggs for scripted metric aggregations {pull}32979[#32979] (issues: {issue}29328[#29328], {issue}31597[#31597])
+* Percentile/Ranks should return null instead of NaN when empty {pull}30460[#30460] (issue: {issue}29066[#29066])
+* Render sum as zero if count is zero for stats aggregation {pull}27193[#27193] (issue: {issue}26893[#26893])
+
+Analysis::
+* Remove `delimited_payload_filter` {pull}27705[#27705] (issues: {issue}26625[#26625], {issue}27704[#27704])
+* Limit the number of tokens produced by _analyze {pull}27529[#27529] (issue: {issue}27038[#27038])
+* Add limits for ngram and shingle settings {pull}27211[#27211] (issue: {issue}25887[#25887])
+
+Audit::
+* Logfile auditing settings remove after deprecation  {pull}35205[#35205]
+* Remove index audit output type {pull}37707[#37707] (issues: {issue}29881[#29881], {issue}37301[#37301])
+
+Authentication::
+* Security: remove wrapping in put user response {pull}33512[#33512] (issue: {issue}32332[#32332])
+* Enhance Invalidate Token API {pull}35388[#35388] (issues: {issue}34556[#34556], {issue}35115[#35115])
```

Comment:
I think #35388 is not a breaking change.  Rather, it should be under "enhancements"